sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -237,6 +237,16 @@ case class ReplaceTableAsSelect(
   }
 }
 
+/**
+ * The logical plan of the CREATE NAMESPACE command that works for v2 catalogs.
+ */
+case class CreateNamespace(
+  catalog: SupportsNamespaces,
+  namespace: Seq[String],
+  ifNotExists: Boolean,
+  comment: Option[String],
+  locationSpec: Option[String],
```

Comment:
Comment and location should be converted to properties and shouldn't be in the v2 logical plan.

Reply:
Done. Thanks.